buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -116,9 +116,12 @@ RUN yum -y update \
   && yum -y install devtoolset-7-gcc-7.3* devtoolset-7-binutils \
   && yum -y install devtoolset-7-gcc-c++-7.3* devtoolset-2-gcc-gfortran-7.3* \
   && yum clean all \
-  && ln -s /opt/rh/devtoolset-7/root/usr/bin/gcc /usr/bin/cc \
-  && ln -s /opt/rh/devtoolset-7/root/usr/bin/gcc /usr/bin/gcc \
-  && ln -s /opt/rh/devtoolset-7/root/usr/bin/g++ /usr/bin/g++
+  && source /opt/rh/devtoolset-7/enable \
+  && env >> /etc/environment \
```

Comment:
I couldn't find anything that claims to read `/etc/environment', although [1] says 'ssh reads ~/.ssh/environment'.

[1] https://linux.die.net/man/1/ssh

Reply:
Yes, however this only applies to whichever user's home directory it is stored in. The `/etc/environment` file applies to all users. Should it be changed to only apply to the `jenkins` user?

Reply:
The file `/etc/environment` is *not* consulted at all as far as I can tell. Adding mappings does not affect any user environment, not via `ssh` and not via `bash -l`. Writing to that file does nothing and should not be present in `Dockerfile`.

Reply:
Try this:
```
FROM centos:centos6.9
RUN echo 'KEITH=WasHere' >> /etc/environment
```
You won't find `KEITH` in the environment.

Reply:
https://hyc-runtimes-jenkins.swg-devops.com/view/Sandbox/job/Colton_docker_test/55/console
It looks like both of them worked, so I will remove the `/etc/environment` lines from the dockerfile